.travis.yml

```diff
@@ -3,31 +3,32 @@ dist: trusty
 language: node_js
 node_js:
   - '10'
-
 addons:
-  apt:
-    sources:
-      - google-chrome
+  chrome: stable
 
 before_install:
-  - npm i -g makeshift && makeshift -s @systelab-tecnologies -r https://registry.npmjs.org
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
 before_script:
+  - npm install -g makeshift && makeshift -s @systelab-tecnologies -r https://registry.npmjs.org
   - npm install -g codecov
   - npm install -g codacy-coverage
   - npm install
-  - ng build
 
 script:
-  - ng test --watch=false --code-coverage
+  - npm run build
```

Comment:
The order is different in systelab-translate

Reply:
Changed!!!!